src/Compilers/CSharp/Portable/Parser/Lexer.cs

```diff
@@ -901,17 +901,6 @@ private void CheckFeatureAvailability(MessageID feature)
                 return;
             }
 
-            string requiredFeature = feature.RequiredFeature();
```

Comment:
Please do not remove this from code from the compiler. It is intended to facilitate experimental code residing in branches of the compiler, even though the product code does not contain experiments.